components/wrap-collection-layout.js

```diff
@@ -45,8 +47,8 @@ function WrapCollectionComponent(Component) {
       associatedMetadata: associatedMetadata,
     });
 
-    const component = addLoadMore(Component, data, associatedMetadata.load_more, props.collection.slug, data);
-
+    const component = addLoadMore(Component, data, associatedMetadata.enable_load_more_button, props.collection.slug, associatedMetadata.limit_stories, associatedMetadata.load_more_stories_count);
+    
     return [addClientSide, addLazyLoad].reduce((c, f) => f(c, associatedMetadata), component);
   }
 }
```

Comment:
Update naming for addLoadMore, addClientSide, addLazyLoad and add a wrapper to addLoadMore for it to follow the pattern